### ISSUE TITLE
[v4] Labels and Headlines in Panel are not shown semi-bold

### ISSUE THREAD

**Issue by marcogeue** [k-next/kirby#5956]
## Description
Labels and Headlines in the Panel are not shown semi-bold on some OS and in some browsers.
This affects especially the `--font-semi`-Variable in the CSS and probably als '--font-bold' as it was discussed on Discord.

**Expected behavior**  
In Discord there were until now the following ideas to solve this:
- set another font-fallback
- change `--font-semi` to 600 and `--font-bold`to 700  weight

**Screenshots**  
![kirby-3-9-7](https://github.com/getkirby/kirby/assets/129186157/63514505-40cd-4686-8b60-b8eddfac2df6)
![kriby-4-0-0-rc](https://github.com/getkirby/kirby/assets/129186157/fd722de4-8a08-4466-997f-91f8b60093a6)

## Your setup
**Kirby Version**  
4.0.0-rc

**Your system (please complete the following information)**
 - Device: Laptop
 - OS: Pop!_OS 22.04 LTS ( Ubuntu based)
 - Browser: Firefox, Chromium
 - localhost

**Comment by ovenum** [comment]
Not sure if this a regression on Kirbys site.
Pop!_OS is using Fira Sans which supplies many font weights. I can only assume that Pop!_OS loads the bare minimum of fonts required by the os and not the complete Fire Sans family. 

The [font repo](https://github.com/pop-os/fonts/tree/master/fira) for Pop!_OS includes plenty of Fira Sans fonts. FiraSans-Medium.otf is included there and should map to a font weight of 500. So in theory it is available.

Maybe this should be taken to Pop!_OS?

**Comment by distantnative** [comment]
@ovenum @ovenum do 500 and 600 work as font weight when setting font-family to `system-ui, sans-serif`? 
Or even `system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`?

**Comment by ovenum** [comment]
Not running Pop!_OS, just looked at the default font stack and what fonts are available. 
@marcogeue are you able to test this on your system and check what font is used as `system-ui` font in your browser?

Looking at the second screenshot, the font looks different from Fira Sans. Fira Sans has an _open_ lower case e, the e used in the screenshots is quite closed. Maybe my research regarding Pop!_OS system font is wrong or the UA uses a different font than the system font.

<img width="718" alt="image" src="https://github.com/getkirby/kirby/assets/47149622/5a218bf1-2677-42f5-8c1b-24980c151e4d">


I think it’s the responsibility of the OS to supply a sufficient font stack that enables fine tuned UI. Pop!_OS might load a minimal font stack for performance reasons which is great when on computer with limited resources but a tradeoff when a fine tuned interface is required.

**Comment by marcogeue** [comment]
At the moment my firefox browser selects Segoe-UI as `system-ui`, but this is after I installed the font myself trying to solve the issue. It's still the same. But I checked it on my second laptop that also uses Pop!_OS and that one does not have the issue. So there all labels are shown like intended with the semi-bold setting.
So I would say we can close this issue, if it really just affects my very specific use case.
Thanks for the replies.
